Updating Raven Notebooks

datw0258 opened the pull request:
-changes filenames to match with what is currently in the shared box

-adds if-else statements to run raven based on operating system (Windows)

(Let me know if there is a mistake in this PR because I see that my fork was behind arfc/pride master at the time of creating this)

datw0258 committed:
Merge pull request #1 from arfc/master

Update 2020-dotson-optimal-sizing to pride

datw0258 committed:
updates to all raven notebooks

-changes filenames to match with what is currently in the shared box
-adds if-else statements to run raven based on operating system (Windows)

samgdotson reviewed (changes requested):
The changes look good, some small corrections
1. It's not clear where ``platform`` is defined in some notebooks. Possibly do that in one of the first cells to make it super clear where and why. 
2. There should be two minor changes in the paths... ``Research`` should be ``research`` and ``2020-dotson-optimal-sizing`` should be ``pride`` to reflect the recent changes to the repository (I know this one is a little annoying). 

Will merge after these changes are made.

datw0258 committed:
updating to match current repository path

-editted path for running raven to be up-to-dat
-made platform import more clear

datw0258 commented:
Got it

samgdotson reviewed (approved):
Awesome, merging now.
